storage/innobase/handler/ha_innodb.cc

```diff
@@ -20990,7 +20990,7 @@ ib_push_warning(
 		buf[MAX_BUF_SIZE - 1] = 0;
 		vsnprintf(buf, MAX_BUF_SIZE - 1, format, args);
 
-		push_warning_printf(
+		push_warning(
 			thd, Sql_condition::WARN_LEVEL_WARN,
 			uint(convert_error_code_to_mysql(error, 0, thd)), buf);
```

Comment:
(also applies to the copy below)
It’s understandable for this file to `vsnprintf` before `push_warning` because `push_warning_printf` doesn’t have a `vprintf` variant.
But it’s interesting that it preferred using C++ `vsnprintf` over beïng consistent with `push_warning_printf` and using `my_vsnprintf`.